views/login.pug

```diff
@@ -2,6 +2,24 @@ extends layout
 
 block content
 
+  div(id="successAlert" class="alert alert-success" role="alert" style="position: fixed; z-index: 2; right: 20px; display: none; top: -100px")
+  div(id="failAlert" class="alert alert-danger" role="alert" style="position: fixed; z-index: 2; right: 20px; display: none; top: -100px")
+
+
+  div(class="modal fade" id="emailModal" tabindex="-1" role="dialog" aria-labelledby="forgotPassword" aria-hidden="true")
+    div(class="modal-dialog" role="document")
+      div(class="modal-content")
+        div(class="modal-header")
+          h5(class="modal-title" id="forgotPassword") Account recovery
+          button(type="button" class="close" data-dismiss="modal" aria-label="Close")
+            span( aria-hidden="true") &times;
+        div(class="modal-body")
+          p Enter the verified email address associated with your account. We will send you a temporary password.
```

Comment:
The password isn't temporary, so it's better to remove this word from all the messages